DigitalLearningSolutions.Web/Views/LearningPortal/Current/MarkActionPlanItemAsComplete.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current
+@using Microsoft.Extensions.Configuration
+@model SetLearningLogItemCompletionDateViewModel
+@inject IConfiguration Configuration

```

Comment:
Don't think we use Configuration on this page, so it can be safely removed.

Reply:
removed